### DISCUSSION TITLE
Multi Region Support

### DISCUSSION THREAD

**Discussion by ugurcemozturk** [awslabs/aws-mysql-jdbc#320] [Q&A] [answered]
Hey,

How to configure the driver/connection string to let it be aware of the multi-regional deployed Aurora MySQL Cluster?

Let's say I have a cluster in 2 regions, so I've got 2 writer endpoints, 1 active and 1 passive.
Is it possible to put both endpoints to the driver so it can handle the failover by itself?

Best.

**Comment by karenc-bq** [comment, marked as answer]
Hi @ugurcemozturk, the AWS JDBC Driver for MySQL does not have multi-az support yet.

**Reply by ugurcemozturk** [reply]
Thank you @karenc-bq . To confirm, is it only for same region, sama AZ, multi-master setups?

I couldn't get the actual point of cluster awareness mentioned for failover scenarios  in the readme .

Thanks!

**Reply by karenc-bq** [reply]
Correct, the driver currently only supports same region and same AZ clusters.
The cluster awareness mentioned in the documentation means whether you establish a connection via the cluster endpoint or an instance endpoint, should the connection fails the driver is able to failover to another instance within the same cluster, in the same region and AZ.

**Reply by ugurcemozturk** [reply]
Thank you!
